### DISCUSSION TITLE
Early Universe Quasars and the Eddington Limit

### DISCUSSION THREAD

**Discussion by MJoseph315** [a-cosmology-group/acg#294] [Astrophysics and Physics]
What’re everyone’s thoughts on this video?
https://www.youtube.com/watch?v=0rzgYzbzq5Q
I feel it’s relevant to this group because it deals with explaining the presence of mature astronomical objects in the early universe.

**Comment by RedshiftDrift** [comment]
The video explains how objects can form faster than the Eddington limit.  Super-Eddington accretion rates 100 times are possible with thin disks, although higher accretion rates would need very thin disks which have never been observed in nature.

But the video does not mention many other related problems.
- There is not enough matter around these quasars to feed the black hole for long enough to reach the large masses obtained.  (Sorry, I forget which papers have measured that...)  In some cases they calculate a required efficiency that is **larger than 100%**, which does not make sense, of course.
- There is also a problem with the number of photons this generates that is too large and incompatible with the CMB! (J. B. Muñoz, J. Mirocha, J. Chisholm, S. R. Furlanetto, and C. Mason, “Reionization after JWST: a photon budget crisis?” Apr. 10, 2024, doi: [10.48550/arXiv.2404.07250](https://doi.org/10.48550/arXiv.2404.07250).)
- It is not understood why these black holes that "supposedly form so rapidly" suddenly stop accumulating matter https://phys.org/news/2024-12-dormant-massive-black-hole-early.html

Everything points to a serious problem with almost everything happening in "the early universe".  It's not only with quasars, but with [galaxy formation](https://thedaily.case.edu/james-webb-space-telescope-results-may-upend-astronomers-theory-of-galaxy-formation/), not enough matter to produce all these objects, not enough time to produce these objects, dark matter clumpiness much higher than thought, a changing cosmological constant as [dark energy is getting weaker](https://www.space.com/desi-cosmological-constant-dark-energy-history), etc. etc. etc.

How likely is it that all these models turn out to be wrong at the same time?  The better explanation is that they all depend on the ***incorrect model*** of an expanding universe with a Big Bang.
